### DISCUSSION TITLE
Influx / Grafana

### DISCUSSION THREAD

**Discussion by deliverer33** [MartinStoffel/tigo#7] [General]
Wondering if you have tried to get this data into influx?  I'm trying to "learn" both right now but struggling with the learning curve of the 2 products.  If you have had any success, maybe I can pick your brain on configuration if I get stuck.  I have it flowing into Influx, but struggling with Grafana.

**Comment by MartinStoffel** [comment]
Hi, I never did that. However, when the data is in the influx DB and you strongly think that your Grafana query is ok, but you don't see data/or can't access the Influx DB then try to enter the IP of your computer instead of 127.0.0.1 or localhost. In addition Grafana might trick you with the range of data queried/displayed. Set this to several days to start with might help

**Reply by deliverer33** [reply]
cool thanks.  maybe once I build this out I can share a template others might be able to utilize. large learning curve so it might take me a bit.

**Reply by deliverer33** [reply]
Is there a definition of the fields somewhere and maybe I missed it.  Example A1_power, A1_reclaimed_power, A1_energy etc...

**Reply by MartinStoffel** [reply]
Not sure if I understand this question. The README.md shows the values and reclaimed power is what Tigo claims to "generate" in addition to what you would get without the optimizers. I hope this answers your queston.
